camel/types/enums.py

```diff
@@ -65,6 +65,9 @@ class ModelType(UnifiedModelType, Enum):
     CLAUDE_3_5_SONNET = "claude-3-5-sonnet-20240620"
 
     # Nvidia models
+    NVIDIA_LLAMA3_CHATQA_70B = "nvidia/llama3-chatqa-1.5-70b"
+    NVIDIA_LLAMA3_CHATQA_8B = "nvidia/llama3-chatqa-1.5-8b"
+    NVIDIA_NEMOTRON_340B = "nvidia/nemotron-4-340b-instruct"
```

Comment:
could we support more popular models,  like `yi-large`, `mistral-large`, `llama 3.1 405b`, `llama 3.1 70b`